layouts/_default/newsletters-signup.html

```diff
@@ -131,9 +131,11 @@ <h2 class="text-xl font-bold lg:text-2xl">
                       {{ if $page.Pages }}
                         <a
                           class="text-blue underline hover:text-blue-7"
-                          href="{{ $page.RelPermalink }}#archives"
+                          href="{{ $page.RelPermalink }}"
```

Comment:
The `if $page.Pages` above this means "only put the link to view the archives on newsletters with archives". We should just give them all links, since we're going to take away the archives.